src/forces/Force.re

```diff
@@ -7,11 +7,7 @@ type entity = {
 
 let applyForce = (~force, ~entity) => {
   // Derive the acceleration created by the force and add it to the current acceleration.
-  let nextAcceleration =
-    Vector.addf(
-      ~v1=entity.acceleration,
-      ~v2=Vector.divf(~v=force, ~s=entity.mass),
-    );
+  let nextAcceleration = Vector.divf(~v=force, ~s=entity.mass);
```

Comment:
The keybug fix – we don't want to accumulate `acceleration`, we just want to derive it on every frame to calculate the `nextVelocity`. Theoretically we don't even need to store it in `state` anymore! Leaving it for now in case it ends up being useful in some way.